servers/v3-proxy-api/codegen.ts

```diff
@@ -5,6 +5,7 @@ const config: CodegenConfig = {
   documents: ['src/**/*.graphql'],
   generates: {
     './src/generated/graphql/types.ts': {
+      config: { federation: true },
```

Comment:
oh interesting, wonder why generation cares

Reply:
It might not be strictly required but it changes some directives